eZ/Publish/Core/Persistence/Cache/ContentHandler.php

```diff
@@ -24,6 +24,7 @@
 class ContentHandler extends AbstractHandler implements ContentHandlerInterface
 {
     const ALL_TRANSLATIONS_KEY = '0';
+    const DEFAULT_LIST_VERSIONS_LIMIT = -1;
```

Comment:
You can remove the constant as we don't use it anymore.
If there where use it should have been private.